### ISSUE TITLE
Bug with Unicode

### ISSUE THREAD

**Issue by wooorm** [Flet/github-slugger#9]
```js
> S = require('github-slugger');
[Function: BananaSlug]
> s = new S
BananaSlug { occurrences: {} }
> s.slug('I ♥ unicode')
'i-♥-unicode'
```

...but, GitHub slugs that to [`i--unicode`](https://github.com/wooorm/gh-and-npm-slug-generation#i--unicode).

Apparently, this used to work fine: `github-slugger` used to output `i--unicode` (compare the [last Travis](https://travis-ci.org/Flet/github-slugger/jobs/125908851#L185) build with the [one I just triggered](https://travis-ci.org/Flet/github-slugger/jobs/212456496#L235)).

It’s rather weird that this changed, I’ve got no idea why...

**Comment by wooorm** [comment]
This is introduced in emoji-regex@6.1.1 in https://github.com/mathiasbynens/emoji-regex/commit/37d8faac5a725e37bb5e9ff9531dd9b241c63ae0, but exposed (as the old code was still used in the npm module) in emoji-regex@6.1.3 (and no, there was no emoji-regex@6.1.2 release).

Not sure whether `♥` is special though...

**Comment by Flet** [comment]
Should we lock down package.json to a specific version of emoji-regex for now?

**Comment by wooorm** [comment]
Yeah I think so!

**Comment by wooorm** [comment]
Oh, and do you have ideas for a fix? I’m thinking an overwrite for this character?

**Comment by Flet** [comment]
published as `github-slugger@1.1.2`
